ogcapi-features/README.md

```diff
@@ -1,42 +1,90 @@
+- **OpenAPI specifications:**
+  - [STAC API - Features](openapi-features.yaml) ([rendered version](https://api.stacspec.org/v1.0.0-rc.1/ogcapi-features))
+  - [STAC API - Collections](openapi-collections.yaml) ([rendered version](https://api.stacspec.org/v1.0.0-rc.1/collections))
+- **Conformance Class URIs:**
+  - <https://api.stacspec.org/v1.0.0-rc.1/ogcapi-features> - Features
+  - <https://api.stacspec.org/v1.0.0-rc.1/collections> - Collections
 - **[Maturity Classification](../README.md#maturity-classification):** Candidate
 - **Dependencies**:
   - [STAC API - Core](../core)
-  - [STAC API - Collections](../collections)
   - [OGC API - Features](https://www.ogc.org/standards/ogcapi-features)
   uses all the OGC API - Features openapi fragments to describe returning STAC Item objects.
 
-Adding OGC API - Features (OAFeat) to a STAC API means fully implementing all its requirements, and then returning STAC
-[Item](../stac-spec/item-spec/README.md) objects from its `/items` endpoints. In OAFeat, GeoJSON is an optional
-conformance class, enabling flexibility. However, STAC requires the use of GeoJSON for OAFeat
-endpoints.
+The *STAC API - Collections and Features* specification extends the
+[OGC API - Features - Part 1: Core](https://docs.opengeospatial.org/is/17-069r3/17-069r3.html)
+(OAFeat) specification. OAFeat
+allows returning any content type from its endpoints, whereas *STAC API - Features* requires support
+for returning STAC [Item](../stac-spec/item-spec/README.md) and STAC [Collection](../stac-spec/item-spec/README.md)
+entities. As these entities are also GeoJSON types, the GeoJSON conformance class of OGC API - Features is supported
+(<http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/geojson>).
+
+## Conformance Classes
+
+While OAFeat defines a single conformance class for its endpoints, STAC API divides these behaviors into two
+conformance classes -- Collections and Features. The STAC API - Features
+(<https://api.stacspec.org/v1.0.0-rc.1/ogcapi-features>) conformance class includes and extends the behavior
+of OAFeat, while the STAC API - Collections (<https://api.stacspec.org/v1.0.0-rc.1/collections>) conformance
+class is the subset of Features that pertains only to Collections.
+
+### STAC API - Features
+
+The *STAC API - Features* (<https://api.stacspec.org/v1.0.0-rc.1/ogcapi-features>) conformance class
+encompasses all of the behavior described in this specification, as
+derived from OAFeat.
+
+### STAC API - Collections
+
+The *STAC API - Collections* (<https://api.stacspec.org/v1.0.0-rc.1/collections>) conformance class
+requires only the subset of the behavior of Features that relates to Collections.
+
+This subset is:
+- the `data` link relation on the landing page
+- the `/collections` and `/collections/{collection_id}` endpoints
+
+This subset of behavior is useful in conjunction
+with the *STAC API - Item Search` conformance class to allow the description of STAC Collections
```

Comment:
I appreciate the guidance, but I don't really understand how it is useful in conjunction
with the *STAC API - Item Search`? Why is it useful in this case?

Reply:
```suggestion
with the *STAC API - Item Search* conformance class to allow the description of STAC Collections
```

Reply:
Without this, there's no place that describes what the collections actually are. This text was already in here, so I'd prefer not to change it with this PR -- if you want to file an issue, we can review later.